test/functional/checkupdate/no-server-content/lines-checked

```diff
@@ -1,2 +1,6 @@
+swupd-client software update checker 3.9.4
+   Copyright (C) 2012-2016 Intel Corporation
+
```

Comment:
You should remove the above three lines, because they match the ignored patterns (see test/functional/ignore-list for the pattern list). In particular, the first line would need modification for every new release of swupd-client so would create a maintenance problem.